pkg/stacks/unpack_test.go

```diff
@@ -1176,7 +1243,10 @@ func TestUnpack(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			got := &bytes.Buffer{}
 			rd := &walker.ResourceDir{Base: tt.root, Walker: afero.Afero{Fs: tt.fs}}
-			err := Unpack(rd, got, tt.root, "Namespaced", "crossplane/ts-controller:0.0.0")
+
+			os.Setenv(StackImageEnv, "crossplane/sample-stack:0.0.1")
+			err := Unpack(rd, got, tt.root, "Namespaced", "crossplane/ts-controller:0.0.0", logging.NewLogrLogger(zap.Logger(true)))
```

Comment:
Unless we are testing that the logger received a log message, I think it is safe to replace `logging.NewLogrLogger(zap.Logger(true))` with `logging.NewNopLogger()`

Reply:
I considered this, but I decided to put a real logger in so that the log output could be used to help debug test failures.